src/css/app.css

```diff
@@ -208,7 +208,7 @@ a:hover {
 }
 
 .identification a {
-  color: white;
+  color: var(--dusk);
```

Comment:
@joshfraser was this intended to dark the user name here or something somewhere else?

![usercard](https://user-images.githubusercontent.com/273937/40630082-17b0f8c6-6295-11e8-9c9a-fabf0b691b49.png)

Reply:
Oops my bad. I changed that class because the buyer & seller names on the purchase view were invisible because they got turned white when linked. Didn't realize that class was used in multiple places.

Reply:
@micahalcorn No, that's here:

<img width="347" alt="screen shot 2018-05-28 at 2 40 02 pm" src="https://user-images.githubusercontent.com/6504519/40630155-0a2f775a-6285-11e8-90c0-d6a85f7cb6b1.png">

Reply:
@tyleryasaka poor naming practice on my part, maybe. I've moved your target link up a couple of generations in the tree. I'll change this back and they will play nice.